### PR TITLE
Add sizing features

### PR DESCRIPTION
**Pull request by jamesnw**
- Adds key to aspect-ratio
- Adds box-sizing
- Adds width-height
- Adds (min | max)-content
- Adds (min | max)-(width | height)
- Adds fit-content, and fit-content()

I did not add `stretch` yet, as there is no support yet.

`min-height:auto` seems to have incorrect support, opened https://github.com/mdn/browser-compat-data/issues/24582

### CODE REVIEW

File: features/min-max-width-height.yml.dist [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  - css.properties.max-height.none
+
+  # baseline: false
+  # support:
+  #   chrome: "21"
+  #   chrome_android: "25"
+  #   edge: "79"
+  #   safari: "7"
+  #   safari_ios: "7"
+  - css.properties.min-height.auto
```
Comment (ddbeck) [inline features/min-max-width-height.yml.dist:92]: I sent https://github.com/mdn/browser-compat-data/pull/24601 to fix these

File: features/fit-content-function.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+name: fit-content()
+description: The `fit-content()` CSS function clamps a size between `min-content` and `max-content`.
```
[the remark this answers is not in the document]
Comment (jamesnw) [reply features/fit-content-function.yml:2]: I agree, `length` is better than `size` here. Thanks!

**Comment by ddbeck** [comment]
@jamesnw I think this is ready to merge, except `min-max-width-height` which is blocked by https://github.com/mdn/browser-compat-data/pull/24601. Do you want me to set this whole PR as blocked, or would you like to break out that one feature?

**Comment by jamesnw** [comment]
> @jamesnw I think this is ready to merge, except `min-max-width-height` which is blocked by [mdn/browser-compat-data#24601](https://github.com/mdn/browser-compat-data/pull/24601). Do you want me to set this whole PR as blocked, or would you like to break out that one feature?

@ddbeck I'll set this whole PR to blocked to limit task overhead.

**Pull request merged**